functions/conf/crowd_to_github_groups.json

```
@@ -51,6 +51,26 @@
       "team": "SDRANAccess"
     }
   ],
+  "SDCoreAccess" : [
+    {
+      "githubOrg": "omec-project",
+      "team": "SDRANAccess"
```

Should this be `SDCoreAccess`? The `SDRANAccess` team is already synchronized with `omec-project` (see above).
```suggestion
      "team": "SDCoreAccess"
```

---

🤦 Fixed.